server/routes/spotsHW.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var uuid = require('uuid');
+var router = express.Router();
+
+var SPOTS_API_KEY;
+if(process.env.BUILD == 'development') {
+  SPOTS_API_KEY = '1234';
+} else {
+  SPOTS_API_KEY = uuid();
+}
+console.log("Using SPOTS_API_KEY = %s", SPOTS_API_KEY)
+
+router.post('/update/:key/:lotID/:spotID/:vacancy/:cardID?', (req, res, next) => {
```

Comment:
This is a dummy method that still needs to get defined. But this is how it will work.

I'm thinking we only need this one. the post will make one call whether someone is checking in or not.